Model/Sales/TrackTraceHolder.php

```diff
@@ -332,14 +330,34 @@ private function getWeightTypeOfOption(?string $weight): int
         return (int) $weight ?: 1000;
     }
 
+    /**
+     * Get country of origin from product settings or, if they are not found, from the MyParcel settings.
+     *
+     * @param $product_id
+     * @return string
+     */
+    public function getCountryOfOrigin($product_id): string
+    {
+        $product = $this->objectManager->get('Magento\Catalog\Api\ProductRepositoryInterface')->getById((int) $product_id);
+        $productCountryOfOrigin = $product->getCountryOfManufacture();
+        $mpCountryOfOrigin = $this->helper->getGeneralConfig('basic_settings/country_of_origin');
```

Comment:
zet dit inline in je return statement, nu doe je alsnog die call terwijl je hem misschien niet gebruikt

Reply:
```suggestion
```